bowling/src/printer.cpp

```diff
@@ -0,0 +1,52 @@
+#include "printer.h"
+
+void Printer::print(std::vector<LaneStruct> lanes) {
+    for(auto & lane : lanes){
+        printHeader(lane);
+        printPlayers(lane);
+    }
+}
+
+void Printer::printHeader(LaneStruct & lane){
+    printText("### " + lane.name + ": " + parseStatus(lane.status) + " ###\n");
```

Comment:
You can exctract `###` to  constexpr. What is more, if you are concatenate more than 2 strings, use `append()` or string stream :)